network-api/app/networkapi/highlights/apps.py

```diff
@@ -6,5 +6,4 @@ class HighlightsConfig(AppConfig):
     verbose_name = 'highlights'
```

Comment:
we should be able to remove this file entirely, if we remove the contents (but not the file) of `network-api/app/networkapi/highlights/__init__.py`